packages/playwright-test/src/dispatcher.ts

```diff
@@ -466,6 +466,7 @@ class Worker extends EventEmitter {
   private _didSendStop = false;
   private _didFail = false;
   private didExit = false;
+  private readonly ready: Promise<void>;
```

Comment:
style:

```suggestion
  private _ready: Promise<void>;
```

Reply:
Fixed in https://github.com/microsoft/playwright/pull/13270/commits/39a0a52c33b816018056ee12f387e3e99674ffa4 and https://github.com/microsoft/playwright/pull/13270/commits/44297b77fdfef941e59e36f6f8b9340bf45308cf